data/common.yaml.tmpl

```diff
@@ -890,8 +836,3 @@ rsyslog::client::server: {{ config.syslog_server }}
 
 # HAproxy
 haproxy_auth: {{ config.haproxy_auth | default('admin:changeme') }}
-
-# This is needed for initial bootstrap
-# before step1 is even generated
-classes:
- - cloud
```

Comment:
Moved to `data/type.yaml.tmpl`, so it avoid step 0 issues on nodes that have no step 1